server/src/db.py

```diff
@@ -1,40 +1,71 @@
-import os
+import logging
 
-from motor import motor_asyncio
+import asyncpg
+from aiohttp import web
+from asyncpg.pool import Pool
 
+from config import BASE_DIR
 
-DB_CONNECTION = None
-DATABASE = None
 
+log = logging.getLogger(__name__)
 
-def db_connection():
-    global DB_CONNECTION
-    if DB_CONNECTION is None:
-        DB_CONNECTION = motor_asyncio.AsyncIOMotorClient(os.environ['MONGO_URL'])
-    return DB_CONNECTION
 
+def get_query(query_name: str) -> str:
+    try:
+        with open(BASE_DIR / f'queries/{query_name}.sql', 'r') as file:
+            return file.read()
+    except FileNotFoundError:
+        log.error(f'Query {query_name} not found')
+        return ''
 
-def db() -> motor_asyncio.AsyncIOMotorDatabase:
-    global DATABASE
-    if DATABASE is None:
-        DATABASE = db_connection()[os.environ['MONGO_DATABASE']]
-    return DATABASE
 
+async def create_db(app: web.Application):
+    log.info('create_db')
+    pool: Pool = app['pool']
+    async with pool.acquire() as con:
+        await con.execute(get_query('init'))
 
-async def create_room(data: dict):
-    await db().rooms.insert_one(data)
 
+async def create_room(pool: Pool, data: dict) -> int:
+    async with pool.acquire() as con:
+        room_id = await con.fetchval(
+            get_query('create_room'),
+            data['room_name'], data['is_private']
+        )
+    if 'password' in data.keys():
+        await update_password(pool, room_id, data['password'])
+    return room_id
 
-async def get_rooms(page: int, limit: int) -> (list, int):
-    cursor = db().rooms.aggregate([
-        {"$limit": limit},
-        {"$skip": (page - 1) * limit},
-    ])
-    return await cursor.to_list(None), await db().rooms.count_documents({})
 
+async def get_room(pool: Pool, data: dict):
+    async with pool.acquire() as con:
+        room = await con.fetchrow(
```

Comment:
You must get room by it's **id**, not name.
Id - is unique identifier, primary key, search by primary key is faster (thanks for [indexes](https://ru.wikipedia.org/wiki/%D0%98%D0%BD%D0%B4%D0%B5%D0%BA%D1%81_(%D0%B1%D0%B0%D0%B7%D1%8B_%D0%B4%D0%B0%D0%BD%D0%BD%D1%8B%D1%85)))